### PR TITLE
Update banned dependencies include.

### PR DESCRIPTION
**Pull request by conniey**
Fixing the metadata for banned-dependencies to use eph's dependency version.

### CODE REVIEW

File: sdk/eventhubs/ci.yml [hunk 1]
```diff
@@ -48,3 +48,9 @@ extends:
       - name: azure-messaging-eventhubs-checkpointstore-blob
         groupId: com.azure
         safeName: azuremessagingeventhubscheckpointstoreblob
+      - name: azure-messaging-eventhubs-track1-perf
```
Comment (alzimmermsft) [inline sdk/eventhubs/ci.yml:51]: You'll want to add these under `AdditionalModules`, similar to this: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/ci.yml#L81

May also need to add these as additional modules to tests.yml

File: sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/ReceiveEventsTest.java [hunk 2]
```diff
@@ -69,13 +79,17 @@ public ReceiveEventsTest(EventHubsReceiveOptions options) {
         return Mono.empty();
     }
 
+    /**
+     * {@inheritDoc}
+     */
```
Comment (alzimmermsft) [inline sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/ReceiveEventsTest.java:84]: Was this triggering a Checkstyle issue? Javadocs are inherited when using `@Override`

**Review by alzimmermsft — APPROVED** [review]

File: sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/ReceiveEventsTest.java (hunk 2 above)
Comment (conniey) [reply sdk/eventhubs/azure-messaging-eventhubs-track1-perf/src/main/java/com/azure/messaging/eventhubs/perf/ReceiveEventsTest.java:84]: Nah. There were a ton of javadoc issues. So I just did whichever I saw. :)

**Pull request merged**